backend-rust/src/bin/ccdscan-api.rs

```diff
@@ -92,6 +127,8 @@ async fn main() -> anyhow::Result<()> {
 
     let (subscription, subscription_listener) =
         graphql_api::Subscription::new(cli.database_retry_delay_secs);
+    let (block_sender, block_receiver) = tokio::sync::watch::channel(None);
+    let block_receiver_health = block_receiver.clone();
```

Comment:
The naming of these channel endpoints seems a bit off, right? :sweat_smile: 
These are about the node status